check-licenses/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
   using: "composite"
   steps:
     - name: "Install Git and clone project"
-      if: inputs.skip-install == 'false'
+      if: ${{ (inputs.skip-install == 'false') && (input.checkout == 'true') }}
       uses: actions/checkout@v3
```

Comment:
Revisit this action... I think you are missing the checkout parameter in the definitions.

Reply:
I.e.:

```yaml
  checkout:
      description: 'Whether to checkout the repository or not'
      default: true
      required: false
      type: boolean
```

Reply:
Done!